queries.sql

```diff
@@ -48,3 +48,13 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- Queries Using JOIN
+
+SELECT name AS animal FROM animals  INNER JOIN owners ON owner_id = owners.id WHERE full_name = 'Melody Pond';
+SELECT animals.name AS animal FROM animals INNER JOIN species ON species_id = species.id WHERE species.name = 'Pokemon';
+SELECT full_name AS owner, name AS animal FROM animals  RIGHT JOIN owners ON owner_id = owners.id ORDER BY full_name;
+SELECT species.name AS species, COUNT(species_id) AS "number of animals" FROM animals INNER JOIN species ON species_id = species.id GROUP BY species.name;
+SELECT animals.name AS Digimons FROM animals INNER JOIN owners ON owner_id = owners.id WHERE full_name = 'Jennifer Orwell' AND animals.name LIKE '%mon'; 
+SELECT animals.name AS animals FROM animals INNER JOIN owners ON owner_id = owners.id WHERE full_name = 'Dean Winchester' AND animals.escape_attempts < 0;

```

Comment:
- [ ] Great job working on the functionality so far but I think here you should have 'Charmander' but you have an empty row as a result so I think it would be great if you updated it and put animals.escape_attempts = 0 🔥😊✅💯✨

Reply:
Thank you @Reem-lab, let me change